### ISSUE TITLE
School of hard knocks/Uncouth quality not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chummer5a/chummer5a#37]
```
First quality ensures a 2 for one point increase for street knowledge skills, 
thats not happening.

Second requires 2 points for a 1 point increase during and after character gen 
for social skills.

Version 5.156.1
```

Original issue reported on code.google.com by `rashidi...@gmail.com` on 5 Mar 2015 at 2:34

**Comment by chummer5a** [comment]
Uncouth is working after chargen.  The chargen version requires some thinking.

**Comment by chummer5a** [comment]
Resolved in 5.162.
